include/rppdefs.h

```diff
@@ -136,7 +136,13 @@ typedef enum
     /*! \brief src and dst layout mismatch \ingroup group_rppdefs */
     RPP_ERROR_LAYOUT_MISMATCH           = -18,
     /*! \brief Number of channels is invalid. (Needs to adhere to function specification.) \ingroup group_rppdefs */
-    RPP_ERROR_INVALID_CHANNELS          = -19
+    RPP_ERROR_INVALID_CHANNELS          = -19,
+    /*! \brief Invalid output tile length (Needs to adhere to function specification.) \ingroup group_rppdefs */
+    RPP_ERROR_INVALID_OUTPUT_TILE_LENGTH    = -20,
+    /*! \brief Shared memory size needed is beyond the bounds (Needs to adhere to function specification.) \ingroup group_rppdefs */
+    RPP_ERROR_OUT_OF_BOUND_SHARED_MEMORY_SIZE    = -21,
+    /*! \brief Scratch memory size needed is beyond the bounds (Needs to adhere to function specification.) \ingroup group_rppdefs */
+    RPP_ERROR_OUT_OF_BOUND_SCRATCH_MEMORY_SIZE    = -22,
```

Comment:
If error code 20, 21, 22 are like 14, pls use the word "insufficient" like 14

Reply:
Checked 14th error code, RPP_ERROR_INSUFFICIENT_DST_BUFFER_LENGTH
It is not like 14

In NSR HIP kernel case, since we are using pre-allocated scratch memory for some intermediate buffers and for a certain input configuration if the required scratch memory goes beyond the bounds, we return this error codes

Where as insufficient are the cases, where the user passed buffers are lesser than required memory
https://github.com/sampath1117/rpp/blob/sr/nsr_hip/src/modules/rppt_tensor_statistical_operations.cpp#L123

So not change the names here